app/qml/components/MMListView.qml

```diff
@@ -8,13 +8,35 @@
  ***************************************************************************/
 
 import QtQuick
+import QtQuick.Controls
 
 //
 // Hot-fix for hotfix https://github.com/MerginMaps/mobile/issues/3417
 // Seems like there is some issue with cache in ListView
 //
 
 ListView {
+  id: root
 
   cacheBuffer: 0
+  readonly property bool isMobile: (Qt.platform.os === "android" || Qt.platform.os === "ios")
+  property int scrollSpace: !isMobile ? 10 : 0
+
+  ScrollBar.vertical: ScrollBar {
+    id: verticalScrollBar
+
+    policy: ScrollBar.AlwaysOn
+    visible: !isMobile && (root.contentHeight > root.height)
+    opacity: (pressed || root.moving) ? 0.7 : 0.4
```

Comment:
Shouldn't `active` be enough https://doc.qt.io/qt-6/qml-qtquick-controls-scrollbar.html#active-prop? 
```suggestion
    opacity: active ? 0.7 : 0.4
```